typescript/src/Vaas.ts

```diff
@@ -215,6 +237,34 @@ export class Vaas {
       ws.send(verdictReq);
     });
   }
+
+  private async forStreamRequest(
+    stream: Readable
+  ): Promise<VaasVerdict> {
+    const ws = this.getAuthenticatedWebSocket();
+    return new Promise((resolve, reject) => {
+      const guid = uuidv4();
+      if (this.debug) console.debug("uuid", guid);
+      this.verdictPromises.set(guid, {
+        resolve: async (verdictResponse: VerdictResponse) => {
+          if (verdictResponse.verdict === Verdict.UNKNOWN) {
+            const contentLength = stream.readableLength;
+            await this.upload(verdictResponse, stream, contentLength);
+            this.verdictPromises.delete(guid);
+          }
+          resolve(new VaasVerdict(verdictResponse.sha256, verdictResponse.verdict));
```

Comment:
Doesn't this mean that if Malicious or Clean comes back, the user gets a Verdict without the content having been seen by the server? This shouldn't happen in theory, but just in case, we throw a VaasServerException here in the other SDKs...